src/main/app/cypress/hakukohdeFormUtils.ts

```diff
@@ -6,27 +6,64 @@ import {
   stubOppijanumerorekisteriHenkiloRoute,
   stubKoodiRoute,
   stubCommonRoutes,
+  jatka,
 } from '#/cypress/utils';
 
-import organisaatio from '#/cypress/data/organisaatio';
 import haku from '#/cypress/data/haku';
 import hakukohde from '#/cypress/data/hakukohde';
 import createKoodi from '#/cypress/data/koodi';
 import koulutus from '#/cypress/data/koulutus';
 import toteutus from '#/cypress/data/toteutus';
 import valintaperuste from '#/cypress/data/valintaperuste';
 
+const toimipisteTarjoajat = [
+  '1.2.246.562.10.16538823663',
+  '1.2.246.562.10.2013081415065445951365',
+  '1.2.246.562.10.20485193278',
+  '1.2.246.562.10.20866993056',
+  '1.2.246.562.10.23017513880',
+  '1.2.246.562.10.45854578546',
+  '1.2.246.562.10.55355715099',
+  '1.2.246.562.10.55711304158',
+  '1.2.246.562.10.56139411567',
+  '1.2.246.562.10.60465978314',
+  '1.2.246.562.10.656909794910',
+  '1.2.246.562.10.78321186062',
+  '1.2.246.562.10.879177258610',
+  '1.2.246.562.10.93115250668',
+];
+
+const selectedToimipisteNimi = /stadin ammatti- ja aikuisopisto, myllypuron toimipaikka/i;
+
+const selectedToimipiste = '1.2.246.562.10.45854578546';
+
+export const fillJarjestyspaikkaSection = ({ jatka: jatkaProp = true }) => {
+  cy.findByTestId('jarjestyspaikkaOidSection').within(() => {
+    toimipisteTarjoajat.forEach(oid => {
+      cy.get(`[value="${oid}"]`).should('exist');
+      if (oid === selectedToimipiste) {
+        cy.get(`[value="${oid}"]`).should('not.be.disabled');
+      } else {
+        cy.get(`[value="${oid}"]`).should('be.disabled');
+      }
+    });
+
+    cy.findByText(selectedToimipisteNimi).click();
+    jatkaProp && jatka();
```

Comment:
Itse vierastan && käyttöä kutsun ehdollisuudessa, e.g.:
https://www.codereadability.com/dont-use-short-circuiting-when-you-want-an-if-statement/
"Using short-circuiting makes code difficult to read. An if statement clearly communicates that the code should only be run if a condition is fulfilled. But using a logical operator for the same purpose is just confusing."

Reply:
Ihan hyvä pointti. Olen itse tullut näille jossain määrin sokeaksi. Pyrin parantamaan tapani tältä osin.